[2022-06-29 20:38:33,543] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T20:38:29.728852+00:00 [queued]>
[2022-06-29 20:38:33,611] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T20:38:29.728852+00:00 [queued]>
[2022-06-29 20:38:33,613] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 20:38:33,615] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-06-29 20:38:33,619] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 20:38:33,680] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): postgreSQL_getTable> on 2022-06-29T20:38:29.728852+00:00
[2022-06-29 20:38:33,727] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'SDG_Enrique_V0', 'postgreSQL_getTable', '2022-06-29T20:38:29.728852+00:00', '--job-id', '1319', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/company_forecasting.py', '--cfg-path', '/tmp/tmpamqdqn0i', '--error-file', '/tmp/tmp4qkuerek']
[2022-06-29 20:38:33,704] {standard_task_runner.py:52} INFO - Started process 265 to run task
[2022-06-29 20:38:33,742] {standard_task_runner.py:77} INFO - Job 1319: Subtask postgreSQL_getTable
[2022-06-29 20:38:34,073] {logging_mixin.py:104} INFO - Running <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T20:38:29.728852+00:00 [running]> on host 1ff8f57a8cbf
[2022-06-29 20:38:34,171] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=enrique-liebana@outlook.com
AIRFLOW_CTX_DAG_OWNER=Enrique
AIRFLOW_CTX_DAG_ID=SDG_Enrique_V0
AIRFLOW_CTX_TASK_ID=postgreSQL_getTable
AIRFLOW_CTX_EXECUTION_DATE=2022-06-29T20:38:29.728852+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-29T20:38:29.728852+00:00
[2022-06-29 20:38:34,193] {base.py:74} INFO - Using connection to: id: postgres_conection. Host: host.docker.internal, Port: 5432, Schema: data_company, Login: airflow, Password: XXXXXXXX, extra: None
[2022-06-29 20:38:51,217] {company_forecasting.py:55} INFO - Backup Company data in : /raw_data/company.csv
[2022-06-29 20:38:51,226] {python.py:118} INFO - Done. Returned value was: None
[2022-06-29 20:38:51,270] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=SDG_Enrique_V0, task_id=postgreSQL_getTable, execution_date=20220629T203829, start_date=20220629T203833, end_date=20220629T203851
[2022-06-29 20:38:51,506] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-29 20:38:51,669] {local_task_job.py:146} INFO - Task exited with return code 0
